apps/gp2-server/src/routes/public/output.route.ts

```diff
@@ -27,6 +27,17 @@ export const outputRouteFactory = (
     },
   );
 
+  outputRoutes.get(
+    '/outputs/:outputId',
+    async (req, res: Response<gp2Model.PublicOutputResponse>) => {
+      const { outputId } = req.params;
+
+      const output = await outputController.fetchById(outputId);
```

Comment:
does this mean that if we know the id of a non public output we can access it with that api?

Reply:
Good catch!! We shouldn't but yes it probably means that yeah. I'll work on the fix now